src/Api/include/Api/Orbit.h

```diff
@@ -35,9 +35,9 @@
 //
 // Integration:
 // To integrate the manual instrumentation API in your code base, simply include this header file
-// and instantiate the "g_orbit_api_v0" global variable in your code. Orbit will automatically
+// and place the ORBIT_API_INSTANTIATE macro in an implementation file. Orbit will automatically
 // deploy and dynamically load liborbit.so into the target process. Orbit will then write the proper
-// function addresses into the "g_orbit_api_v0" table.
+// function addresses into the "g_orbit_api_v[N]" table.
```

Comment:
```suggestion
// function addresses into the "g_orbit_api_v<N>" table.
```
Maybe a weird suggestion, but it looked like an array otherwise.